torch/_inductor/codecache.py

```diff
@@ -1253,9 +1293,9 @@ def load(cls, source_code, func_name, key, cuda):
                     log.debug("Cpp wrapper done building %s", filepath)
                 else:
                     log.debug("Found target .so, cpp wrapper loading %s", filepath)
-                    spec = importlib.util.spec_from_file_location(name, filepath)
+                    spec = importlib.util.spec_from_file_location(name, filepath)  # type: ignore[attr-defined]
```

Comment:
The linter is telling me that importlib.util doesn't even exist. So.... maybe this code path is not being exercised any more?